uint8 ID
uint8[2] cnt_data 